test/internet/test-tls-autoselectfamily-servername.js

```diff
@@ -15,9 +15,9 @@ setDefaultAutoSelectFamilyAttemptTimeout(common.platformTimeout(common.isWindows
 // Test that TLS connecting works without autoSelectFamily
 {
   const socket = connect({
-    host: 'google.com',
+    host: 'nodejs.org',
```

Comment:
Maybe the user overridable `INET_HOST` from `common/internet`? https://github.com/nodejs/node/blob/bcebb914f59e1713d9044d58fee49622a139a67b/test/common/internet.js#L8